plugins/linuxv2/ifplugin/linuxcalls/admin_linuxcalls.go

```diff
@@ -0,0 +1,63 @@
+//  Copyright (c) 2018 Cisco and/or its affiliates.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at:
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// +build !windows,!darwin
+
+// Copyright (c) 2017 Cisco and/or its affiliates.
```

Comment:
why there are two headers?

Reply:
This file was just copied from the original linux plugin - somebody must have put it there by mistake.

Reply:
You should rebase on latest pantheon-dev probably.. I think this was fixed by @VladoLavor already